client/src/components/APIDataTypes.ts

```diff
@@ -107,11 +107,11 @@ export type SearchResults = {
 
 // TYPES ASSOCIATED WITH SUMMARY AGGREGATE QUERY:
```

Comment:
Does this comment still hold? Maybe `types associated with summary aggregation`, since it's not a query anymore?